### PR TITLE
Add Skills section with sliders that move on page load

### PR DESCRIPTION
**Pull request by haoyxc**
Create a section for skills with a bar that fills up as the page first loads and add Skills to the header for navigation. 

There's also a little flag that shows the % of the skill (out of 100). (Note the skills are not finalized and still are subject to change!!)

What it looks like: 
Screenshot 2020-05-29 at 11 55 07 AM

(I had to zip the video because it was a .webm file and Github wouldn't taken that extension.) This shows how the loading works -- sorry it's a bit long the only important part is what happens after I refresh the page at 4-5ish seconds. I couldn't figure out how to crop it my bad!
[capture.zip](https://github.com/haoyxc/step/files/4703139/capture.zip)

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,26 @@ function setActiveTab(btn) {
   btn.classList.add('active'); 
 }
 
+/**
+ * Load skills bar in skills container
+ */
+function loadSkills () {
+  $(".skill-per").each(function() {
+  let $this = $(this);
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js]: nit: indent this function body

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -116,6 +116,65 @@
...
+  background-color: #000;
+  color: #fff;
+  font-size: 12px;
+  border-radius: 4px;
+  top: -35px;
+  right: 0;
+  transform: translateX(50%);
+}
+
+.skill-per::after {
```
Comment (EatingW) [inline portfolio/src/main/webapp/style.css]: Was there a reason to add an ::after selector with no content?

**Review by EatingW — COMMENTED** [review]
This looks great! 

One thing I noticed from your video demo is that at 0:03 the final value of the percentages are already displayed briefly before resetting to 0% and incrementing. Maybe set them to 0 initially as well? Feel free to do this as a follow up PR.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (haoyxc) [reply portfolio/src/main/webapp/style.css]: OH my bad I think I had something there and then didn't need it. Will remove!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -31,21 +31,21 @@ function setActiveTab(btn) {
...
+          $this.attr("per", Math.floor(this.animatedValue) + "%");
+        },
+        complete: function() {
+          $this.attr("per", Math.floor(this.animatedValue) + "%");
+        }
       }
-    }
-  );
+    );
 });
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js]: Really nit: Also indent this line :)

**Review by EatingW — APPROVED** [review]

**Pull request merged**